src/CoreShop/Bundle/PimcoreBundle/Resources/config/services.yml

```diff
@@ -11,8 +11,15 @@ parameters:
 
 services:
     _defaults:
+        autowire: true
+        autoconfigure: true
         public: true
 
+    CoreShop\Bundle\PimcoreBundle\Controller\Admin\:
```

Comment:
within the coreshop-core we don't use auto-configuration/auto-wiring

Reply:
auto-wire needs to be done for the Admin controller, added only for this controller in `src/CoreShop/Bundle/PimcoreBundle/Resources/config/services/dynamic_dropdown.yml`